go/client/cmd_chat_api.go

```diff
@@ -402,20 +389,20 @@ func (c *CmdChatAPI) sendV1(ctx context.Context, arg sendArgV1) Reply {
 			return c.errReply(err)
 		}
 		rlimits = append(rlimits, ncres.RateLimits...)
-		conversation = ncres.Conv.Info
+		convTriple, convID = ncres.Conv.Info.Triple, ncres.Conv.Info.Id
 	case 1:
-		conversation = existing[0].Info
+		convTriple, convID = existing[0].Metadata.IdTriple, existing[0].Metadata.ConversationID
 	default:
 		return c.errReply(fmt.Errorf("multiple conversations matched"))
 	}
 
 	postArg := chat1.PostLocalArg{
-		ConversationID: conversation.Id,
+		ConversationID: convID,
 		MessagePlaintext: chat1.NewMessagePlaintextWithV1(chat1.MessagePlaintextV1{
 			ClientHeader: chat1.MessageClientHeader{
-				Conv:        conversation.Triple,
-				TlfName:     conversation.TlfName,
-				TlfPublic:   conversation.Visibility == chat1.TLFVisibility_PUBLIC,
+				Conv:        convTriple,
+				TlfName:     *arg.convQuery.TlfName,
```

Comment:
Yes; sorry for the confusion. Let say the user does:

```
{
  "method": "send",
  "params": {
    "options": {
      "conversation_id": 12345678,
      "message": {
        "body": "my credit card no. is 123456"
      }
    }
  }
}
```

There's no way `client` or `service` can confirm with user the TLF of the channel the user is trying to send to. A malicious server could return a `chat1.Conversation` for wrong TLF, and the client code will just happily encrypt the message for that TLF happily